apps/api/src/app/controllers/resources/term.controller.ts

```diff
@@ -62,8 +92,29 @@ export class TermController {
 
     @ApiBearerAuth('JWT')
     @UseGuards(OptionalJwtAuthGuard)
-    @Get('')
+    @Post('')
     async fetchMany(@Request() req, @Body() userQueryOptions?: UserQueryOptions) {
+        const { filter } = userQueryOptions || {};
+
+        if (filter) {
+            const schema = getCoscradDataSchema(TermViewModel);
+
+            const allPaths = extractPathsFromUserFilter(filter);
+
+            const result = allPaths.flatMap((path) =>
+                validateFieldPathForCoscradModel(path, schema)
```

Comment:
We'll want this in the middleware at somepoint. Probably registered as part of the `@ResourceIndexEndpoint` decorator.